### ISSUE TITLE
Issue on page /containers/tutorials/k8s-setup.html - grpc error on inf2 - inference model server

### ISSUE THREAD

**Issue by elygerou** [aws-neuron/aws-neuron-sdk#745]
Hi team,
I followed the steps on the article and run the model in inf2.
And the inference is not working I am always getting grpc error as below when I run the inference model server:

 python3 tensorflow-model-server-infer.py                                                       
2023-09-20 08:40:01.667451: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
Traceback (most recent call last):
  File "tensorflow-model-server-infer.py", line 32, in <module>
    result = stub.Predict(request)
  File "/home/ec2-user/.local/lib/python3.7/site-packages/grpc/_channel.py", line 1161, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/home/ec2-user/.local/lib/python3.7/site-packages/grpc/_channel.py", line 1004, in _end_unary_response_blocking
    raise _InactiveRpcError(state)  # pytype: disable=not-instantiable
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
        status = StatusCode.UNAVAILABLE
        details = "failed to connect to all addresses; last error: UNAVAILABLE: ipv4:35.166.159.80:8500: Socket closed"
        debug_error_string = "UNKNOWN:failed to connect to all addresses; last error: UNAVAILABLE: ipv4:35.166.159.80:8500: Socket closed {grpc_status:14, created_time:"2023-09-20T08:40:03.458141018+00:00"}"
>
Any hints that can help?
Thanks,
Elpida

**Comment by aws-rhsoln** [comment]
Thank you for reporting the issue. To replicate the issue, can you help us with package versions, neuron plugin and device extension versions? Also, can you share the Dockerfile for us to fully replicate the issue. We did add a few fixes in the recent 2.13.2 release, so you can try by updating the packages.
Steps to fix

1. update the neuron driver on the host if possible (version 2.12.14)
2. update the neuron device plugin by setting the image in their k8s-neuron-device-plugin.yml to public.ecr.aws/neuron/neuron-device-plugin:2.16.18.0 and redeploying the device plugin
3. update the k8s neuron scheduler by setting the image in their k8s-neuron-scheduler-eks.yml to public.ecr.aws/neuron/neuron-scheduler:2.16.18.0. Then
```
kubectl apply -f k8s-neuron-scheduler-eks.yml
kubectl apply -f https://awsdocs-neuron.readthedocs-hosted.com/en/latest/_downloads/e739253083129abeaf6f6ad1db7ccb21/my-scheduler.yml
```

**Comment by elygerou** [comment]
Hi Rahul,

thank you for feedback (I also pinged you on Slack we can do a troubleshooting session together).
I am running on inf2.8xlarge and ami is for EKS with GPU: ami-078e3447baec5acfc, by running neuron-top I see the version is: neuron-top 2.14.6.0,  aws-neuronx-tools.x86_64 0:2.14.6.0-1 is installed currently. (If more info required let me know what command to run on the host). So I guess I should not downgrade to 2.12.14?
Then the neuron plugin is public.ecr.aws/neuron/neuron-device-plugin:2.16.18.0 and had run the neuron plugin already per the link.
I had not run the scheduler previously but I did now per the file you attached.
However still facing same issue,
Thanks,
Elpida
